sql/core/pom.xml

```diff
@@ -111,6 +111,19 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.spotify</groupId>
+      <artifactId>docker-client</artifactId>
+      <classifier>shaded</classifier>
+      <version>2.7.7</version>
```

Comment:
Woah, this is an ancient version of `docker-client`. I'll try using a newer release to see if that magically fixes the ASM issues.